bindings/python/src/ser.rs

```diff
@@ -174,6 +177,34 @@ impl Serialize for SerializePyObject {
                     sequence.end()
                 }
             }
+            ObjectType::Tuple => {
```

Comment:
This is copy pasted from the `ObjectType::List` option the only change is `PyTuple_GET_SIZE` vs `PyList_GET_SIZE` and `GET_ITEM`

Is there a better way to merge these into one ?